tests/qemu/qemu.pm

```diff
@@ -15,19 +15,31 @@ use warnings;
 use base "consoletest";
 use testapi;
 use utils;
+use transactional qw(trup_call check_reboot_changes);
+use Utils::Architectures qw(is_x86_64 is_ppc64le is_s390x is_aarch64);
+use version_utils qw(is_sle_micro);
 
+sub is_qemu_preinstalled {
+    return is_sle_micro && check_var('FLAVOR', 'MicroOS-Image') && !script_run 'rpm -q patterns-microos-kvm_host';
```

Comment:
This checks for it directly anyway, so why not just:

```
    return !script_run 'rpm -q patterns-microos-kvm_host';
```

or remove the `rpm -q` if the goal is to ensure that it's preinstalled on that flavor.

Reply:
Or do the `rpm -q` in `install_qemu` for early exit

Reply:
> or remove the `rpm -q` if the goal is to ensure that it's preinstalled on that flavor.

Yup, this is the main goal as it is specified in kiwi and required to have the pattern pre-installed for SUMA.

Reply:
Then it should probably do something like

```
if (is_sle_micro && check_var('FLAVOR', 'MicroOS-Image')) {
    assert_script_run('rpm -q patterns-microos-kvm_host qemu ...');
}
```

Reply:
> Then it should probably do something like
> 
> ```
> if (is_sle_micro && check_var('FLAVOR', 'MicroOS-Image')) {
>     assert_script_run('rpm -q patterns-microos-kvm_host qemu ...');
> }
> ```

Even better, thanks!